ledger-core/virtual/integration/deactivate_test.go

```diff
@@ -139,3 +149,254 @@ func TestVirtual_DeactivateObject(t *testing.T) {
+
+					runnerMock := logicless.NewServiceMock(ctx, t, func(execution execution.Context) string {
+						return execution.Request.CallSiteMethod
+					})
+					server.ReplaceRunner(runnerMock)
+
+					server.Init(ctx)
+					server.IncrementPulseAndWaitIdle(ctx)
+
+					var (
+						object    = reference.NewSelf(server.RandomLocalWithPulse())
+						prevPulse = server.GetPulse().PulseNumber
+					)
+
+					server.IncrementPulseAndWaitIdle(ctx)
+					Method_PrepareObject(ctx, server, payload.Inactive, object, prevPulse)
+
+					gotResult := make(chan struct{})
+
+					typedChecker := server.PublisherMock.SetTypedChecker(ctx, mc, server)
+					typedChecker.VCallResult.Set(func(res *payload.VCallResult) bool {
+
+						assert.Equal(t, res.Callee, object)
+						contractErr, sysErr := foundation.UnmarshalMethodResult(res.ReturnArguments)
+						require.Equal(t, &foundation.Error{callTypeTest.errorMsg}, contractErr)
+						require.NoError(t, sysErr)
+
+						gotResult <- struct{}{}
+
+						return false // no resend msg
+					})
+
+					pl := payload.VCallRequest{
+						CallType:            callTypeTest.callType,
+						CallFlags:           payload.BuildCallFlags(contract.CallIntolerable, stateTest.objectState),
+						Caller:              server.GlobalCaller(),
+						Callee:              object,
+						CallSiteDeclaration: testwallet.GetClass(),
+						CallSiteMethod:      "MyFavorMethod",
+						CallOutgoing:        reference.NewSelf(object.GetLocal()),
+						Arguments:           insolar.MustSerialize([]interface{}{}),
+					}
+					server.SendPayload(ctx, &pl)
+
+					commontestutils.WaitSignalsTimed(t, 10*time.Second, gotResult)
+
+					mc.Finish()
+				})
+			}
+		})
+	}
+}
+
+// 1. Create object
+// 2. Deactivate object
+// 3. Send request on Dirty state - get error
+// 4. Send request on Validated state - get response
+func TestVirtual_CallMethod_On_DeactivatedDirtyState(t *testing.T) {
+	t.Log("C5470")
+	defer commontestutils.LeakTester(t)
+
+	mc := minimock.NewController(t)
+
+	server, ctx := utils.NewUninitializedServer(nil, t)
+	defer server.Stop()
+
+	runnerMock := logicless.NewServiceMock(ctx, t, func(execution execution.Context) string {
+		return execution.Request.CallSiteMethod
+	})
+	server.ReplaceRunner(runnerMock)
+
+	server.Init(ctx)
+	server.IncrementPulseAndWaitIdle(ctx)
+
+	var (
+		object           = reference.NewSelf(server.RandomLocalWithPulse())
+		runnerResult     = []byte("123")
+		deactivateMethod = "deactivatingMethod"
+		prevPulse        = server.GetPulse().PulseNumber
+	)
+
+	server.IncrementPulseAndWaitIdle(ctx)
+	{
+		// Create object
+		Method_PrepareObject(ctx, server, payload.Ready, object, prevPulse)
+	}
+
+	isolation := contract.MethodIsolation{Interference: contract.CallTolerable, State: contract.CallDirty}
+	{
+		// execution mock for deactivation
+		descr := descriptor.NewObject(object, server.RandomLocalWithPulse(), server.RandomGlobalWithPulse(), makeRawWalletState(initialBalance), false)
```

Comment:
don't use wallet in integration tests

Reply:
I understand, what you are talking about

Reply:
`makeRawWalletState(initialBalance)` -> `State: []byte("some memory")`